core/dbt/adapters/cache.py

```diff
@@ -270,14 +268,14 @@ def _add_link(self, referenced_key, dependent_key):
         if referenced is None:
             return
         if referenced is None:
-            dbt.exceptions.raise_cache_inconsistent(
-                "in add_link, referenced link key {} not in cache!".format(referenced_key)
+            raise CacheInconsistency(
+                f"in add_link, referenced link key {referenced_key} not in cache!"
```

Comment:
I left `CacheInconsistency` passing in the message to show how we need to reuse the exception when there are different messages.